src/server/pfs/server/api_server.go

```diff
@@ -629,11 +629,38 @@ func (a *apiServer) DeleteAll(ctx context.Context, request *types.Empty) (respon
 
 // Fsckimplements the protobuf pfs.Fsck RPC
 func (a *apiServer) Fsck(request *pfs.FsckRequest, fsckServer pfs.API_FsckServer) (retErr error) {
-	if err := a.driver.fsck(fsckServer.Context(), request.Fix, func(resp *pfs.FsckResponse) error {
+	ctx := fsckServer.Context()
+	if err := a.driver.fsck(ctx, request.Fix, func(resp *pfs.FsckResponse) error {
 		return errors.EnsureStack(fsckServer.Send(resp))
 	}); err != nil {
 		return err
 	}
+
+	if request.ZombieTarget != nil {
+		return a.driver.detectZombie(ctx, request.ZombieTarget, fsckServer.Send)
+	}
+	if request.FullZombieCheck {
+		// list meta repos as a proxy for finding pipelines
+		a.driver.listRepo(ctx, false, pfs.MetaRepoType, func(info *pfs.RepoInfo) error {
```

Comment:
No error check here?

Reply:
Sorry for the sloppiness on these, there must be an off-the-shelf linting option for this